src/client/models/tankShot.js

```diff
@@ -1,35 +1,38 @@
 'use strict';
 import paper from 'paper';
-import constans from './constans';
+import externalVariables from './externalVariables';
+import externalFunctions from './externalFunctions';
 import showChatWindow from './chatField';
 import tankMovement from './tankMovement';
 
-var findLinePoints = tankMovement.findLinePoints;
-var originalPoints = constans.originalPoints;
+const findLinePoints = tankMovement.findLinePoints,
+    tankMove = tankMovement.tankMove,
+    getId = externalFunctions.getId,
+    requestAnimFrame = externalFunctions.requestAnimFrame;
+
+let originalPoints = externalVariables.originalPoints,
+    tankX = externalVariables.tankObj.tankX,
+    tankY = externalVariables.tankObj.tankY,
+    angleWeapon = externalVariables.tankObj.angleWeapon;
+
+const WIDTH = externalVariables.WIDTH,
```

Comment:
wow!
import externalVariables from './externalVariables'; could be replaced by
import { originalPoints, ...  }from './externalVariables';